README.md

```diff
@@ -101,26 +102,27 @@ await insert(db, {
   description: 'Experience immersive sound quality with these noise-cancelling wireless headphones.',
   price: 99.99,
   meta: {
-    rating: 4.5
-  }
+    rating: 4.5,
+  },
 })
 
 await insert(db, {
   name: 'Smart LED Bulb',
   description: 'Control the lighting in your home with this energy-efficient smart LED bulb, compatible with most smart home systems.',
   price: 24.99,
   meta: {
-    rating: 4.3
-  }
+    rating: 4.3,
+  },
 })
 
 await insert(db, {
   name: 'Portable Charger',
-  description: 'Never run out of power on-the-go with this compact and fast-charging portable charger for your devices.',
+  description:
+    'Never run out of power on-the-go with this compact and fast-charging portable charger for your devices.',
```

Comment:
This is still on a new line

Reply:
It was a different line. Sob. Fixing it.

Reply:
Done